staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/helpers_test.go

```diff
@@ -27,37 +27,6 @@ var (
 	c APIServiceConditionType = "C"
 )
 
-func TestGetAPIServiceConditionByType(t *testing.T) {
```

Comment:
why is it removed?

Reply:
because it's not referenced anywhere

Reply:
it looks that function GetAPIServiceConditionByType is still present, but its test is removed here.

Reply:
it looks that function GetAPIServiceConditionByType is still present, but its test is removed here.